bgpd/bgp_attr.c

```diff
@@ -1647,7 +1647,7 @@ int bgp_mp_reach_parse(struct bgp_attr_parser_args *args,
 #define BGP_MP_REACH_MIN_SIZE 5
 #define LEN_LEFT	(length - (stream_get_getp(s) - start))
 	if ((length > STREAM_READABLE(s)) || (length < BGP_MP_REACH_MIN_SIZE)) {
-		zlog_info("%s: %s sent invalid length, %lu", __func__,
+		zlog_info("%s sent invalid length, %lu, of MP reach NLRI",
```

Comment:
I see we have log structure somewhere like `X.X.X.X sent invalid`, somewhere `X.X.X.X, MP nexthop`, somewhere `(X.X.X.X) Wrong multiprotocol`. I personally would love having unified format for all the logs we have.

Reply:
Thanks, @ton31337, for suggesting this.
I've corrected all of the log messages in `bgp_mp_reach_parse()`.
I'll recheck the log messages for every subsequent PRs that is raised to address logging issues.